File: specs/master/cluster/cluster_stats/cluster_indices_stats.ts

/**namespace:Cluster.ClusterStats */
interface cluster_indices_stats {
	completion: completion_stats;
	count: long;
	docs: doc_stats;
	fielddata: fielddata_stats;
	percolate: percolate_stats;
	query_cache: query_cache_stats;
	segments: segments_stats;
	shards: cluster_indices_shards_stats;
	store: store_stats;
}

File: output/csharp/Cluster/ClusterStats/ClusterIndicesStats.cs
using Nest.CommonOptions;
using Nest.Internal;
using Nest.Cluster;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Cluster {

	public class ClusterIndicesStats  {
		
		[DataMember(Name="completion")]
		public CompletionStats Completion { get; set; }


		[DataMember(Name="count")]
		public long Count { get; set; }


		[DataMember(Name="docs")]
		public DocStats Docs { get; set; }


		[DataMember(Name="fielddata")]
		public FielddataStats Fielddata { get; set; }


		[DataMember(Name="query_cache")]
		public QueryCacheStats QueryCache { get; set; }


		[DataMember(Name="segments")]
		public SegmentsStats Segments { get; set; }


		[DataMember(Name="shards")]
		public ClusterIndicesShardsStats Shards { get; set; }


		[DataMember(Name="store")]
		public StoreStats Store { get; set; }

	}
}
